apps/api/public/dashboard.html

```
@@ -1302,4 +1196,4 @@ <h3 class="text-xl font-bold text-white mb-2">Audit Logs coming soon</h3>
 </html>
 </body>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </body> ]

(tag-pair)